platform/android/display_server_android.h

```diff
@@ -116,6 +118,9 @@ class DisplayServerAndroid : public DisplayServer {
 	virtual String clipboard_get() const override;
 	virtual bool clipboard_has() const override;
 
+	virtual Error file_dialog_show(const String &p_title, const String &p_current_directory, const String &p_filename, bool p_show_hidden, const FileDialogMode p_mode, const Vector<String> &p_filters, const Callable &p_callback) override;
+	void emit_file_picker_callback(bool p_ok, const Vector<String> &p_selected_paths, int p_filter);
```

Comment:
The `Error` constant is also define in Android; let's use that instead of using `bool` values.

Reply:
`bool` is used by other platform too, so it's for consistency. FileDialog class also uses this callback and expects boolean.